docs/neural_modeling/plot_06_calcium_imaging.py

```diff
@@ -91,38 +114,43 @@
 plt.show()
 
 # %%
-# The downsampling did not destroy the transient dynamic. We can now move on to using nemos to fit a model.
+# The downsampling did not destroy the fast transient dynamics, so seems fine to use. We can now move on to using nemos to fit a model.
 
 # %%
 # ## Basis instantiation
 # 
-# We can define a cyclic-BSpline for capturing the encoding of the heading angle, and a
-# log-spaced raised cosine basis for the coupling filters. We can combine the two basis.
+# We can define a cyclic-BSpline for capturing the encoding of the heading angle, and a log-spaced raised cosine basis for the coupling filters between neurons. Note that we are not including a self-coupling (spike history) filter, because in practice we have found it results in overfitting.
+#
+# We can combine the two bases.
 
 heading_basis = nmo.basis.CyclicBSplineBasis(n_basis_funcs=12)
 coupling_basis = nmo.basis.RaisedCosineBasisLog(3, mode="conv", window_size=10)
 
 # %%
-# We need to make sure the design matrix will be full-rank by applying identifiability constraints
-# to the Cyclic Bspline
-heading_basis.identifiability_constraints = True
+# We need to make sure the design matrix will be full-rank by applying identifiability constraints to the Cyclic Bspline, and then combine the bases (the resturned object will be an `AdditiveBasis` object).
 
-# Here we combine the basis. The returned object is an `AdditiveBasis` object.
+heading_basis.identifiability_constraints = True
 basis = heading_basis + coupling_basis
 
+
 # %%
 # ## Gamma GLM
 #
-# Since the transients are non-negative, we will use a Gamma distribution from `nemos` with a soft-plus non linearity.
-#
+# Until now, we have been modeling spike trains, and have used a Poisson distribution for the observability model. With calcium traces, things are quite different: we no longer have counts but continuous signals, so the Poisson is no longer appropriate. We cannot use a Gaussian because the calcium traces are non-negative. To satisfy these constraints, we will use a Gamma distribution from `nemos` with a soft-plus non linearity.
```

Comment:
```suggestion
# Until now, we have been modeling spike trains, and have used a Poisson distribution for the observability model. With calcium traces, things are quite different: we no longer have counts but continuous signals, so the Poisson is no longer appropriate. A a Gaussian model is not ideal since the calcium traces are non-negative. To satisfy these constraints, we will use a Gamma distribution from NeMoS with a soft-plus non linearity.
```